src/cr/cube/dimension.py

```diff
@@ -1019,58 +1040,62 @@ def label(self) -> str:
 
         This value is the empty string when no value has been specified or display of
         the name has been suppressed.
+        """
+        return self._retreive_value("name")
 
-        This property handles elements for variables of all types, including
+    @lazyproperty
+    def missing(self) -> bool:
+        """True if this element represents missing data.
+
+        False if this category or subvariable represents valid (collected)
+        data.
+        """
+        return bool(self._element_dict.get("missing"))
+
+    @lazyproperty
+    def numeric_value(self) -> Union[int, float]:
+        """Numeric value assigned to element by user, np.nan if absent."""
+        numeric_value = self._element_dict.get("numeric_value")
+        return np.nan if numeric_value is None else numeric_value
+
+    def _retreive_value(self, key: str) -> str:
```

Comment:
* retrieve

Reply:
`_retrieve_value` is not very informative. I'd prefer a different name. Like, what are we actually trying to do here? It used to be a `label`, but it was too complicated for just the label. So it's more like "textual representation of an element". So maybe something like `_as_string()`, or something similar. Or `stringify`, or something like that. I know these are not perfect, but it tells us that the intended value is `str`. Or maybe, if you wanted to be as explicit as possible, then you'd do something like `get_str_representation_for(key="name")`...

Reply:
yeah seems a very good point